engine/org.eclipse.birt.report.engine/src/org/eclipse/birt/report/engine/layout/pdf/util/HTML2Content.java

```diff
@@ -639,6 +642,7 @@ static void handleElement(Element ele, Map<Element, StyleProperties> cssStyles,
 			childCell.setColumn(1);
 			childCell.setColSpan(1);
 			childCell.setInlineStyle(style);
+			childCell.setTagType("LBody");
```

Comment:
Suggestion: Make it sense to create constants on an interface for all specialized PDF/UA tag types, may be at your interface ITagType?

Reply:
That's a good idea. But I don't find the time ATM.

Reply:
I suggest to do this refactoring later.